src/Projects.js

```diff
@@ -1,73 +1,40 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import { List, Icon } from 'semantic-ui-react';
-import { withRouter } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 
 import './Projects.css';
 
-const api_key = 'keyQBC5qtKpZy4cWf';
-const table = 'Labs Project Tracking Staging';
-const view = 'All Projects';
-const projects_uri = `https://api.airtable.com/v0/app1f3lv9mx7L5xnY/${table}?view=${view}&api_key=${api_key}`;
-
 class Projects extends Component {
-  constructor(props) {
-    super(props)
-
-    this.state = {
-      projects: []
-    }
-  }
-
-  componentDidMount() {
-    this.fetchProjectsData();
-  }
-
-  fetchProjectsData() {
-    return fetch(projects_uri)
-      .then((response) => {
-        return response.json();
-      })
-      .then((response) => {
-        let projects = response.records.map((record) => { return record.fields; });
-        this.setState({
-          projects
-        });
-      });
-  }
-
-  linkTo() {
-    withRouter(({ history }) => {
-      return history.push('/ideas/1');
-    });
-  }
-
   render() {
-    let projects = () => {
-      let projects = this.state.projects;
-      return projects.map((project, index) =>         
-        <List.Item
-          as='a'
-          onClick={this.linkTo}
-          key={index}>
-          <Icon 
-            name="idea" />
+    const getProjects = () => {
+      const { projects } = this.props;
+      return projects.map(d => (
+        <List.Item key={d.project_id} >
+          <Icon name="idea" />
           <List.Content>
-            { project.project_name } - 
-            { project.point_of_contact }
+            <Link to={`/ideas/${d.project_id}`}>
+              { d.project_name } - { d.point_of_contact }
+            </Link>
           </List.Content>
         </List.Item>
-      );
-    }
+      ));
+    };
 
     return (
       <List
-        size='huge'
+        size="huge"
```

Comment:
This is another airbnb lint thing, strings in js code should be single quoted, but strings as props should be double quoted, as should HTML tag properties.